packages/backend/src/services/DashboardService/DashboardService.ts

```diff
@@ -287,6 +287,71 @@ export class DashboardService {
         return this.getById(user, dashboardUuid);
     }
 
+    async updatePinning(
+        user: SessionUser,
+        dashboardUuid: string,
+        dashboard: UpdateDashboard,
```

Comment:
We can make this argument stricter since this method is just to update the pinning. 
And with that we can remove the code about adding a version and the `isDashboardUnversionedFields` check